core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -270,7 +271,8 @@ object ConfigCommand extends Config {
     }
     if (props.containsKey(LogConfig.MessageFormatVersionProp)) {
       println(s"WARNING: The configuration ${LogConfig.MessageFormatVersionProp}=${props.getProperty(LogConfig.MessageFormatVersionProp)} is specified. " +
-        s"This configuration will be ignored if the version is newer than the inter.broker.protocol.version specified in the broker.")
+        "This configuration will be ignored if the version is newer than the inter.broker.protocol.version specified in the broker or " +
```

Comment:
Should we also mention that the configuration is deprecated? Same question `TopicCommand` and the warning message in `LogConfig`.

Reply:
Sure, sounds good.